### PR TITLE
Adjust the functional architecture diagram and documents

### PR DESCRIPTION
**Pull request by luobily**
#### What type of this PR
Adjust the functional architecture diagram and documents

Add one of the following kinds:
/kind document

#### Specified Reviewers:

/assign @Effet @sfwn

### CODE REVIEW

**Review by sfwn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![Erda logo](./docs/assets/logo-small.jpg)
 
-## An enterprise-grade application building, deploying, monitoring platform
+## An enterprise-grade Cloud-Native application platform.
```
Comment (sfwn) [inline README.md:3]: May be no `.` at the end is better ?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sfwn) [reply README.md:3]: @Dreamica

Comment (Dreamica) [reply README.md:3]: Agree with @sfwn.
`.` is unnecessary and “cloud-native” should be lowercase.

**Pull request merged**